third_party/move/tools/move-lint/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+pub mod lint;
+use std::path::PathBuf;
+
+use codespan::{FileId, Files};
+use codespan_reporting::diagnostic::Diagnostic;
+
+pub fn move_lint(path: PathBuf) -> (Vec<Diagnostic<FileId>>, Files<String>) {
+    lint::main(path)
```

Comment:
This gives me a compile error - I cannot test any code because of this.

Reply:
refactored